src/components/ProjectsList.astro

```diff
@@ -30,7 +30,11 @@ for (const project of projects) {
 {projects.map(async (project) => {
   return (
     <section>
-      <header>{project.title}</header>
+      {project.link ? (
+        <a href={project.link}><header>{project.title}</header></a>
```

Comment:
I was about to ask you to change this to put the link inside the `<header/>` rather than wrapping it, but I've just looked it up properly and it seems since HTML5 it's entirely legitimate to do this and is more-or-less neutral for SEO, even (insofar as that matters in this instance, which is, of course, not very far anyway).  Block-level links are allowed; TIL...

Reply:
OK cool, I actually was wondering which arrangement is (more) blessed. Good to know!

Reply:
Yeah -- it still looks and feels a bit wrong to me, but it's legal.  The practical difference is that the whole header is a click target this way, which in this case is a good thing.  (I would still probably write it the other way around and use `display: inline-block` on the `<a/>` tag, I think, but that's just old habits at this point.)